### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#137] [Esercizi - Bertini] [answered]
![Es 1](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/024a4f1e-eef1-4e27-8dcb-de7e74ec54d3)

**Comment by daromacarlo** [comment]
![Immagine 2023-10-14 164247](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/135441131/c87add1e-bbef-4fd9-82cd-c3ef87e8e9da)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/8ccaf22a-1c5c-412a-ace1-d62d71fa07eb)
Analogo a @CarloDaRomadev

**Comment by Elia-Belli** [comment, marked as answer]
- Numeriamo gli iscritti da 1 a 25, sia $w_p$ il presidente e $w_s$ il segretario:
 $\Omega = \\{(w_p,w_s)\ w_p,w_s = 1,...,25, w_p\neq w_s\\}$
 $|\Omega |= \frac{25!}{(25-2)!}=25\cdot 24$

- $A = \\{singolo\ iscritto\ ricopre\ una\ carica\\}$
 $A_1=\\{singolo\ iscritto\ è\ presidente\\}, |A_1|=1$
 $A_2=\\{singolo\ iscritto\ è\ segretario\\}, |A_2|=1$
 Siccome un iscritto non può essere sia presidente che segretario, $A_1,A_2$ sono eventi disgiunti, allora $A = A_1 \cup A_2$
 $\mathbb{P}(A) = \mathbb{P}(A_1) + \mathbb{P}(A_2) = 1/25 + 1/25 = 2/25$
